addons/tmf_loadouts/loadouts/US_Rangers_2020.hpp

```diff
@@ -0,0 +1,657 @@
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"B_Carryall_mcamo"};
+	secondaryWeapon[] = {"ace_csw_m3CarryTripodLow"};
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {
+		"ACE_Vector"
+	};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"CFP_Kitbag_MCam_Grn"};
+	secondaryWeapon[] = {"CUP_launch_MAAWS"};
+	secondaryAttachments[] = {"cup_optic_maaws_scope"};
+	magazines[] += {
+		LIST_2("CUP_MAAWS_HEDP_M"),
+		"CUP_MAAWS_HEAT_M"
+	};
+};
+
+class matac : r {
+	displayName = "MAT Ammo Carrier";
+	backpack[] = {"CFP_Kitbag_MCam_Grn"};
+	backpackItems[] = {
+		LIST_2("CUP_MAAWS_HEAT_M"),
+		"CUP_MAAWS_HEDP_M"
+	};
+};
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	backpack[] = {"CFP_Kitbag_MCam_Grn"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"ACE_Vector"
+	};
+	backpackItems[] = {
+		LIST_2("CUP_MAAWS_HEAT_M"),
+		"CUP_MAAWS_HEDP_M"
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backpack[] = {"B_Carryall_mcamo"};
+	secondaryWeapon[] = {"CUP_launch_Javelin"};
+	magazines[] += {"CUP_Javelin_M"};
+};
+
+class hatac : r {
+	displayName = "HAT Ammo Carrier";
+	backpack[] = {"B_Carryall_mcamo"};
+	backpackItems[] = {"CUP_Javelin_M"};
+};
+
+class hattl : car {
+	displayName = "HAT Team Leader";
+	backpack[] = {"B_Carryall_mcamo"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"ACE_Vector"};
+	backpackItems[] = {"CUP_Javelin_M"};
+};
+
+class mtrg : car {
+	displayName = "Mortar Gunner";
+	secondaryWeapon[] = {"ace_csw_staticMortarCarry"};
+	backPack[] = {"B_Carryall_mcamo"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrac : r {
+	displayName = "Mortar Ammo Carrier";
+	backPack[] = {"B_Carryall_mcamo"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { MTR_GEAR };
+	backPack[] = {"B_Carryall_mcamo"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class samg : car {
+	displayName = "AA Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+};
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	backPack[] = {"CFP_Camelbak_Mule_Drab"};
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+	linkedItems[] += {"Rangefinder"};
+};
+
+class sn : r {
+	displayName = "Sniper";
+	vest[] = {"CUP_V_CPC_light_mc"};
+	backpack[] = {"B_Carryall_mcamo"};
+	hmd[] = {"CUP_NVG_GPNVG_black"};

```

Comment:
No ghillies, or at least a boonie hat or such?

Reply:
Nope, US Ranger Snipers don't run around in Ghillie suits. They run around in normal (but slick) kits - references: https://imgur.com/a/hXuuUTw

Reply:
chicken bullied me into giving them ghillies